code/game/turfs/simulated.dm

```diff
@@ -71,6 +71,17 @@
 							step(H, H.dir)
 							to_chat(H, "<span class='notice'>You tripped over your hair!</span>")
 
+			//A billowing floor-length cloak is a tripping hazard
+			if(H.alphas.len > 0)
```

Comment:
What's the issue?
I put this check in because I figured a single number comparison would be better to do on every mob every time they move rather than looping through everything in all their hands, since it's uncommon for mobs to have their alphas list changed.